content/build/development-guides/adding-an-environment-to-the-portability-service.md

```diff
@@ -0,0 +1,25 @@
+---
+path: "/build/development-guides/adding-an-environment-to-the-portability-service.md"
+date: "2018-07-12"
+title: "Adding an Environment to the Portability Service"
+---
+
+## Why add an environment to the portability service?
+
+HCA pipelines and other user-submitted pipelines are executed in multiple environments during development and release to demonstrate their portability. The environments in which these pipelines run benefit in two ways:
+
+1. Pipeline developers get quick feedback about limitations they may be introducing into their pipelines that make them incompatible with a particular environment, so they are more likely to write pipelines that execute successfully in the tested environments.
+2. Execution environment implementers get feedback about what kinds of pipelines do or do not work in their environments. With this information, they can identify potential issues with or enhancements to their environments.
+
+
+## Adding an execution environment
+
+The portability service only makes workflow execution requests using the [GA4GH WES API](https://github.com/ga4gh/workflow-execution-service-schemas). So the first step to adding an environment is to make sure that the environment implements WES.
```

Comment:
I'll add a link for WES (from that GA4GH repo README) and maybe a sentence that explains it